Gaijinizer patch/CommonEvents_316.txt

```diff
@@ -139,36 +150,39 @@ have no trouble. Aren't you making a lot?#MTLed
 目先の快楽に集中できて、自分に正直になれて
  いろんなことから、開放的になれるんだ…♡
 > CONTEXT: CommonEvents_316/136/Dialogue/enemy11
-But it is not necessary to think about such
-things. I can concentrate on the pleasures at the
-end and be honest with myself. From various
-things, I can be open-minded...♡#MTLed
+But if they lived without thinking so much, they
+could focus on the immediate pleasures at hand,
+become honest with themselves, and live free and
+open-minded... ♡#Ron
```

Comment:
Flow: (Let's break up this really long sentence)
>_**If they only lived without thinking so much...
They could focus on the immediate pleasures at hand,
be honest with themselves, and live more freely
and open-minded... ♡**_